pingdom-checks/patient-generated-data.ping

```diff
@@ -0,0 +1,11 @@
+# -*- sh -*-
+
+pingdom save-check \
+  --template fhir-resource \ 
+  -a name=sandbox-pgd-r4-questionnaire \ 
+  -a host=sandbox-api.va.gov \
+  -a url="/services/fhir/pgd/v0/r4/questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$SLACK_API_ALARMS_PGD,$SLACK_VASDVP_MONITORING" 
```

Comment:
pull this list out into a separate var, like: https://github.com/department-of-veterans-affairs/lighthouse-external-monitoring/blob/master/pingdom-checks/data-query.ping#L3